start-from-zero/Javascript/src/views/evaluation/evaluation.js

```diff
@@ -1,24 +1,24 @@
 function EvaluationsPage(options){
     const headings = ['Name','Language','Level',''];
-    const rows = [
-        {
-            name:'Dan Constantinescu',
-            language: 'Javascript',
-            level:'Junior'
-        },
-        {
-            name:'Adolf Hgg',
-            language: 'SwS',
-            level:'Master'
-        },
-        {
-            name:'Fiodor D',
-            language: 'Russian',
-            level:'Master'
-        },
-    ];
+    const rows = JSON.parse(localStorage.getItem('evaluations'));
+    // [
```

Comment:
please remove commented code you no longer need

Reply:
Ok. Sorry about this.